media/filters/chunk_demuxer.h

```diff
@@ -39,12 +42,18 @@ class MEDIA_EXPORT ChunkDemuxerStream : public DemuxerStream {
  public:
   using BufferQueue = base::circular_deque<scoped_refptr<StreamParserBuffer>>;
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  ChunkDemuxerStream(const std::string& mime_type,
+                     Type type,
+                     MediaTrack::Id media_track_id);
+#else   // BUILDFLAG(USE_STARBOARD_MEDIA)
   ChunkDemuxerStream() = delete;
 
   ChunkDemuxerStream(Type type, MediaTrack::Id media_track_id);
 
   ChunkDemuxerStream(const ChunkDemuxerStream&) = delete;
```

Comment:
This is for disallowing copy and assign. Is there any reason we want to disable them?